core/shared/src/main/scala/zio/Schedule.scala

```diff
@@ -1126,6 +1127,40 @@ object Schedule {
     Schedule(loop(None, 0L))
   }
 
+  /**
+   * Cron-like schedule that recurs every specified `minute` of each hour.
+   * It triggers at zero second of the minute.
+   * Producing a count of repeats: 0, 1, 2.
+   */
+  def minuteOfHour(minute: Minute): Schedule[Any, Any, Long] = {
+
+    def loop(n: Long): StepFunction[Any, Any, Long] =
+      (now: OffsetDateTime, _: Any) =>
+        ZIO.succeed(
+          Decision.Continue(
+            n + 1,
+            nextFixedOffset(now, minute.value, ChronoField.MINUTE_OF_HOUR).withSecond(0),
+            loop(n + 1L)
+          )
+        )
+
+    Schedule(loop(0L))
+  }
+
+  /**
+   *  Value must be in range [0 ... 59]
+   *  Throws [[java.lang.IllegalArgumentException]] on invalid values.
+   */
+  case class Minute(value: Int) {
```

Comment:
I would not introduce this. It doesn't really belong in ZIO and it has the same problem, which is that if you call it "outside ZIO", you can't deal with the error inside ZIO.